### ISSUE TITLE
BL1850B 5.0Ah replaced the cells, and it stopped showing the charge on the indicator.

### ISSUE THREAD

**Issue by Vovan-sar** [mnh-jansson/open-battery-information#24]
Hello. They brought a BL1850B 5.0Ah battery, it was working, but it was quickly discharged. I replaced the cells, and it stopped showing the charge on the indicator.
I suppose it got locked. I found your development and decided to try to remove the lock with it. But I failed (((
The 240080 chip was not mentioned anywhere, but I decided to try anyway, because I had an Arduino Nano at hand.
I apologize if there are any inaccuracies, I used Google Translate.
![No Interface](https://github.com/user-attachments/assets/e2bfccc7-2230-4232-bc6b-c45f0cf4bfb8)
![Chip 240080](https://github.com/user-attachments/assets/0dc95bd9-cd01-4252-9646-c87fc1e78d49)

**Comment by mnh-jansson** [comment]
Hi, this should be supported. Try to follow the instructions on the README, for both OpenBatteryInformation and ArduinoOBI,

Once you have connected an interface you should read an interface version, this indicates that everything was set up properly

**Comment by Vovan-sar** [comment]
I can't check it anymore, I've already given the battery away. I made arrangements with a friend who has a charger and a load (a screwdriver) and decided to check it.
As soon as I inserted it into the charger, the battery came to life, and it accepts the charge and the indicator started working. And it works in the load too.
Although I couldn't read it, but still, thank you very much for the program, I think it will help many people!!!
